docs/Workflow.md

```diff
@@ -132,13 +138,13 @@ git push -f origin myfeature
 
 1. Visit your fork at https://github.com/$user/onnx-mlir (replace `$user` obviously).
 2. Click the `Compare & pull request` button next to your `myfeature` branch.
+3. Please provide a meaningful description of the pull request's new functionality.
+4. If your pull request is not currently ready for review, we recommend to activate the `draft mode` button. Pull requests that are failing some CIs should probably remain in draft mode.
 
 ### Step 9: Get a code review
 
-Once your pull request has been opened, it will be assigned to at least one
-reviewer. The reviewer(s) will do a thorough code review, looking for
-correctness, bugs, opportunities for improvement, documentation and comments,
-and style.
+Once your pull request has been opened and is not in draft mode anymore, it will be assigned to at least one reviewer. 
```

Comment:
This is not quite the case. It is the contributor of the pull request that have to select one or more reviewers. It does not happen automatically.